datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -53,6 +53,14 @@ enum EntityType {
     The MLModel Entity
     """
     MLMODEL
+    """
+    The DATAFLOW Entity
+    """
+    DATAFLOW
+    """
+    The DATAFLOW Entity
+    """
+    DATAJOB
 }
 
 type Query {
```

Comment:
Should we also support fetch DataJob by urn?

Reply:
yes, definitely!